examples/webgl_materials_physical_transmission.html

```diff
@@ -99,37 +99,18 @@
 					depthWrite: false,
```

Comment:
This should no longer be needed here. Right?

Reply:
That's right 👍 https://github.com/mrdoob/three.js/commit/95df9f8bbc3a7aa9e3d67951d353f14f6143c87c